src/player/Player.js

```diff
@@ -49,32 +35,47 @@ class Player {
   }
 
   render() {
-    updateSpeedometer(this.vx, this.ax)
+    updateSpeedometer(this.vx, this.ax, this.vy)
     this.vx += this.ax
     this.x += this.vx
+    this.y += this.vy
     if (rocket.velocity < 7) {
       rocket.x = this.x + rocketDefaultX
     }
 
+    if (rocket.velocity < 7) {
```

Comment:
There is the same `if ()` above.
No need to duplicate it.
Use one `if` for both actions.